app/src/Controller/UsersController.php

```diff
@@ -60,7 +63,7 @@ public function add()
         $this->render('form');
     }
 
-    /**
+     /**
```

Comment:
Does the space was auto added or accidentally added? 

From cs check, it's complain the error as follow:

```
 66 | ERROR | [x] Doc block not aligned with code; expected indentation of 4 but found 5
```